src/actions/market.js

```diff
@@ -377,6 +382,7 @@ export const sellMarketShares = (market, share, outcomeTokenCount, earnings) =>
 
   // Reset the allowance if the cost of current transaction is greater than the current allowance
```

Comment:
These comments (also next line) are out of place, you can check how it's done in Olympia: https://github.com/gnosis/olympia-interface/blob/master/src/actions/market.js#L279